src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1533,18 +1220,21 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         return getFieldValue(oppPaymentRecord, PARENT_OPPORTUNITY_FIELD);
     }
 
-    @wire(getObjectInfo, {objectApiName: OPP_PAYMENT_OBJECT.objectApiName})
+    @wire(getObjectInfo, { objectApiName: OPP_PAYMENT_OBJECT })
     oppPaymentObjectInfo;
 
-    @wire(getObjectInfo, {objectApiName: OPPORTUNITY_OBJECT.objectApiName})
+    @wire(getObjectInfo, { objectApiName: OPPORTUNITY_OBJECT })
     opportunityObjectInfo;
 
-    @wire(getObjectInfo, {objectApiName: ACCOUNT_OBJECT.objectApiName})
+    @wire(getObjectInfo, { objectApiName: ACCOUNT_OBJECT })
     accountObjectInfo;
 
-    @wire(getObjectInfo, {objectApiName: CONTACT_OBJECT.objectApiName})
+    @wire(getObjectInfo, { objectApiName: CONTACT_OBJECT })
     contactObjectInfo;
 
+    @wire(getObjectInfo, { objectApiName: DATA_IMPORT_OBJECT })
```

Comment:
Just a note, we might be able to take advantage of the new `getObjectInfos` and replace all these individual adapters in the future.

https://developer.salesforce.com/docs/component-library/documentation/en/lwc/reference_wire_adapters_object_infos

Reply:
Yep, great call!  We should start using this for new components going fwd and swap these individual ones out in a future WI.